test_data/ontologies/freetest-onto.ttl

```diff
@@ -1,181 +1,270 @@
+                                                                                                owl:minCardinality
+                                                                                                        "0"^^xsd:nonNegativeInteger ;
+                                                                                                salsah-gui:guiOrder
+                                                                                                        "3"^^xsd:nonNegativeInteger ],
+                                                                                              [ rdf:type
+                                                                                                        owl:Restriction ;
+                                                                                                owl:onProperty
+                                                                                                        :hasInteger ;
+                                                                                                owl:minCardinality
+                                                                                                        "0"^^xsd:nonNegativeInteger ;
+                                                                                                salsah-gui:guiOrder
+                                                                                                        "4"^^xsd:nonNegativeInteger ],
+                                                                                              [ rdf:type
+                                                                                                        owl:Restriction ;
+                                                                                                owl:onProperty
+                                                                                                        :hasIntegerProperty ;
+                                                                                                owl:minCardinality
+                                                                                                        "0"^^xsd:nonNegativeInteger ;
+                                                                                                salsah-gui:guiOrder
+                                                                                                        "5"^^xsd:nonNegativeInteger ] ;
+    knora-base:resourceIcon                                                                   "thing.png" ;
+    rdfs:label                                                                                "FT de"@de, "FT fr"@fr,
+                                                                                              "FT it"@it, "FT en"@en ;
+    rdfs:comment                                                                              """A comment for FT."""@de .
+
+
```

Comment:
Can this part be improved regarding formatting?

Reply:
As discussed, we are going to solve the formatting in another PR/task.